docs/api/rasa-sdk.rst

```diff
@@ -0,0 +1,90 @@
+:desc: Information about `rasa-sdk` objects and functions.
+
+.. _rasa-sdk:
+
+Rasa SDK
+========
+
+.. edit-link::
+
+Rasa SDK provides the tools you need to write :ref:`custom-actions`.
+
+.. contents::
+   :local:
+
+``Action``
+----------
+
+The ``Action`` class is the base class for any custom action. It has two methods
+that both need to be overwritten, `.name()` and `.run()`.
+
+.. _custom_action_example_verbose:
+
+In a restaurant bot, if the user says "show me a Mexican restaurant",
+your bot could execute the action ``ActionCheckRestaurants``,
+which might look like this:
+
+.. testcode::
+
+   from rasa_sdk import Action
+   from rasa_sdk.events import SlotSet
+
+   class ActionCheckRestaurants(Action):
+      def name(self) -> Text:
+         return "action_check_restaurants"
+
+      def run(self,
+              dispatcher: CollectingDispatcher,
+              tracker: Tracker,
+              domain: Dict[Text, Any]) -> List[Dict[Text, Any]]:
+
+         cuisine = tracker.get_slot('cuisine')
+         q = "select * from restaurants where cuisine='{0}' limit 1".format(cuisine)
+         result = db.query(q)
+
+         return [SlotSet("matches", result if result is not None else [])]
+
+
+You should add the the action name ``action_check_restaurants`` to
+the actions in your domain file. The action's ``run`` method receives
+three arguments. You can access the values of slots and the latest message
+sent by the user using the ``tracker`` object, and you can send messages
+back to the user with the ``dispatcher`` object, by calling
+``dispatcher.utter_template``, ``dispatcher.utter_message``, or any other
+``rasa_sdk.executor.CollectingDispatcher`` method.
```

Comment:
"by calling `dispatcher.utter_message`" now!

Maybe we can actually put more information about `utter_message` and all of the different keyword arguments it takes? could even do it via automethod probably

Reply:
Updated that :P 
But automethod does not seem to get the latest rasa-sdk, at least locally even if I pip install the sdk from a local copy 😅